File: backend/app/models/inquiry.py
from datetime import datetime


def new_inquiry(
    name: str,
    email: str,
    message: str,
    phone: str | None = None,
    package_id: str | None = None,
    event_date: datetime | None = None,
    event_time: str | None = None,
    event_duration: str | None = None,
) -> dict:
    return {
        "name": name,
        "email": email,
        "phone": phone,
        "package_id": package_id,
        "message": message,
        "event_date": event_date,
        "event_time": event_time,
        "event_duration": event_duration,
        "status": "new",
        "created_at": datetime.utcnow(),
    }

File: src/models/inquiry.ts
export function newInquiry(
  name: string,
  email: string,
  message: string,
  phone: string | null = null,
  packageId: string | null = null,
  eventDate: Date | null = null,
  eventTime: string | null = null,
  eventDuration: string | null = null,
) {
  return {
    name,
    email,
    phone,
    package_id: packageId,
    message,
    event_date: eventDate,
    event_time: eventTime,
    event_duration: eventDuration,
    status: 'new',
    created_at: new Date(),
  };
}
